### DISCUSSION TITLE
Track Selection on MCP

### DISCUSSION THREAD

**Discussion by rcaia** [helgoboss/helgobox#782] [Ideas]
When I select tracks with ReaLearn that normally expand from a narrow form to wider form when selected by mouse click the tracks highlight but do not physically expand.  If track was previously selected by mouse [and expanded] then Realearn only partially collapses the track collapsing the graphics but not the physical width.  I can't find a setting to modify this behavior... is there a wat to fix?

Thanks much.

**Comment by helgoboss** [comment]
There seem to be different ways to select a track in REAPER and ReaLearn uses a pretty "minimal" way. I'll convert this into a feature request. I remember there was one other issue that's related.
